cem-react-wrapper.config.js

```diff
@@ -16,7 +16,12 @@ import { resolve } from 'path';
 import yaml from 'js-yaml';
 
 export default {
-    globs: ['**/sp-*.ts', '**/src/[A-Z]*.ts'],
+    globs: [
+        '**/sp-*.ts',
+        '**/active-overlay.ts',
```

Comment:
`<active-overlay>` isn't an element I would expect a consumer to ever leverage directly. I'm not sure we should include it here. Have you run into a specific case where you wanted to use it?